packages/vm/src/buildBlock.ts

```diff
@@ -242,7 +245,11 @@ export class BlockBuilder {
 
       // Guard against the case if a tx came into the pool without blobs i.e. network wrapper payload
       if (blobTx.blobs === undefined) {
-        throw new Error('blobs missing for 4844 transaction')
+        // TODO: verify if we want this, do we want to allow the block builder to accept blob txs without the actual blobs?
+        // (these must have at least one `blobVersionedHashes`, this is verified at tx-level)
```

Comment:
What is the context for this?  Do we ever build blocks with the versioned hash but no blob or is that specific to t8n?

Reply:
Should maybe update the comment here. This is if we allow to build a block in the block builder where the blob itself is not available, but the `blobVersionedHashes` in the 4844 tx is. (This is the case in t8ntool, the hashes are there, the actual data is not there. Note that this does not really matter, since EVM does not have access to this data anyways)

Reply:
Ah, ok, then yes, this should be fine (though I agree that we should only allow this behind a guard so it's not run "in production" this way).